src/App/Model.elm

```diff
@@ -80,19 +80,18 @@ type alias Flags =
 
 
 type alias Model =
-    { currentPage : Page
-    , flags : Flags
-    , pages :
-        { home :
-            { promotedArticles : WebData (List Article)
-            , promotedRecipes : WebData (List Recipe)
-            , recipes : WebData (List Recipe)
-            }
-        , articles : WebData (List Article)
-        , recipes : Dict String (WebData (List Recipe))
-        , recipe : { recipe : WebData Recipe, recipes : WebData (List Recipe) }
-        , contact : ContactForm
+    { flags : Flags
+    , currentPage : Page
+    , pageHome :
```

Comment:
I still think it should be divided to small chunks, so this would end up being `pageHome : Home.Model.Model`